react-blocks/src/blocks/Columns/Columns.js

```diff
@@ -0,0 +1,188 @@
+import { React } from 'react';
+import {Component,Fragment} from "@wordpress/element";
+import {
+  TextControl,
+  TextareaControl,
+  ServerSideRender,
+  Dashicon,
+  Tooltip,
+  ToggleControl
+} from '@wordpress/components';
+import {MediaPlaceholder} from "@wordpress/editor";
+
+import {LayoutSelector} from '../../components/LayoutSelector/LayoutSelector';
+import {Preview} from '../../components/Preview';
+
+export class Columns extends Component {
+    constructor(props) {
+      super(props);
+    }
+
+    renderEdit() {
+      const {__} = wp.i18n;
+
+      const {columns_title,columns_description,columns_block_style,columns} = this.props;
+
+      return (
+        <Fragment>
+          <h3>{__('What style of column do you need?', 'p4ge')}</h3>
+
+          <div>
+            <LayoutSelector
+              selectedOption={columns_block_style}
+              onSelectedLayoutChange={this.props.onSelectedLayoutChange}
+              options={[
+                {
+                  label: __('No Image', 'p4ge'),
+                  image: window.p4ge_vars.home + 'images/columns-no_images.jpg',
+                  value: 'no_image',
+                  help: __('Optional headers, description text and buttons in a column display.', 'p4ge')
+                },
+                {
+                  label: __('Tasks', 'p4ge'),
+                  image: window.p4ge_vars.home + 'images/columns-tasks.jpg',
+                  value: 'tasks',
+                  help: __('Used on Take Action pages, this display has ordered tasks, and call to action buttons.', 'p4ge')
+                },
+                {
+                  label: __('Icons', 'p4ge'),
+                  image: window.p4ge_vars.home + 'images/columns-icons.jpg',
+                  value: 'icons',
+                  help: __('For more static content, this display has an icon, header, description and text link.', 'p4ge')
+                },
+                {
+                  label: __('Images', 'p4ge'),
+                  image: window.p4ge_vars.home + 'images/columns-images.jpg',
+                  value: 'image',
+                  help: __('For more static content, this display has an image, header, description and text link.', 'p4ge')
+                },
+              ]}
+            />
+          </div>
+          <div>
+            <TextControl
+              label={__('Title', 'p4ge')}
+              placeholder={__('Enter block title', 'p4ge')}
+              value={columns_title}
+              onChange={this.props.onTitleChange}
+            />
+            <TextareaControl
+              label={__('Description', 'p4ge')}
+              placeholder={__('Enter block description', 'p4ge')}
+              value={columns_description}
+              onChange={this.props.onDescriptionChange}
+            />
+          </div>
+          <div>
+
+            {columns.map((item, index) => {
+              return (
+                <div>
```

Comment:
Hey S! This `<div>` should include a `key` property as: `<div key={index}>`, to avoid a typical "no key in iteration" error, all iterations should include this `key` in the root element.